paymentsheet/src/main/java/com/stripe/android/paymentsheet/state/PaymentSheetLoader.kt

```diff
@@ -100,6 +105,7 @@ internal class DefaultPaymentSheetLoader @Inject constructor(
                 shippingDetails = paymentSheetConfiguration.shippingDetails,
                 hasCustomerConfiguration = paymentSheetConfiguration.customer != null,
                 sharedDataSpecs = sharedDataSpecsResult.sharedDataSpecs,
+                externalPaymentMethodSpecs = externalPaymentMethodSpecs
```

Comment:
Can you add a test for this?

Reply:
There's currently no way to pass an external payment method to paymentSheetLoader.load, so I've held off on adding tests for this function. 

When I add external payment methods to the PaymentSheet.Configuration, then I can write a PaymentSheetLoader test which gets extenral payment methods from the configuration and verify that the correct specs are passed through to PaymentMethodMetadata. 

I have a note on [this jira](https://jira.corp.stripe.com/browse/MOBILESDK-1949) to add those tests